### ISSUE TITLE
Unknown timezone error 

### ISSUE THREAD

**Issue by caggursoy** [kuzmoyev/google-calendar-simple-api#154]
**Describe the bug**

When getting the events from a Google calendar, a "pytz.exceptions.UnknownTimeZoneError" is thrown.

**To Reproduce**

```
from gcsa.google_calendar import GoogleCalendar
from gcsa.event import Event 
gcal = GoogleCalendar('google_email') 
for event in gcal: 
        print(event) 
```

**Expected behavior**

To print each calendar event from the calendar

**Console output**

```
2022-11-24 10:30:00+01:00 - Materialcheck meeting 
2019-11-12 10:30:00+01:00 - Department meeting 
2022-11-15 10:00:00+01:00 - Monday meeting 
2022-11-30 10:30:00+01:00 - Materialcheck / AppTeamMeeting 
Traceback (most recent call last): 
  File "C:\Local\LocalRepos\phdSideQuests\resmanToGCal\main.py", line 50, in <module> 
    main() 
  File "C:\Local\LocalRepos\phdSideQuests\resmanToGCal\main.py", line 10, in main 
    for event in gcal: 
  File "C:\Users\cagatay.guersoy\.conda\envs\env\lib\site-packages\gcsa\_services\events_service.py", line 106, in get_events 
    yield from self._list_events( 
  File "C:\Users\cagatay.guersoy\.conda\envs\env\lib\site-packages\gcsa\_services\events_service.py", line 53, in _list_events 
    yield from self._list_paginated( 
  File "C:\Users\cagatay.guersoy\.conda\envs\env\lib\site-packages\gcsa\_services\base_service.py", line 35, in _list_paginated 
    yield serializer_cls(item_json).get_object() 
  File "C:\Users\cagatay.guersoy\.conda\envs\env\lib\site-packages\gcsa\serializers\event_serializer.py", line 21, in __init__ 
    super().__init__(event) 
  File "C:\Users\cagatay.guersoy\.conda\envs\env\lib\site-packages\gcsa\serializers\base_serializer.py", line 17, in __init__ 
    self.obj = self.to_object(obj) 
  File "C:\Users\cagatay.guersoy\.conda\envs\env\lib\site-packages\gcsa\serializers\base_serializer.py", line 45, in to_object 
    return cls._to_object(json_) 
  File "C:\Users\cagatay.guersoy\.conda\envs\env\lib\site-packages\gcsa\serializers\event_serializer.py", line 140, in _to_object 
    return Event( 
  File "C:\Users\cagatay.guersoy\.conda\envs\env\lib\site-packages\gcsa\event.py", line 159, in __init__ 
    self.start = ensure_localisation(self.start, timezone) 
  File "C:\Users\cagatay.guersoy\.conda\envs\env\lib\site-packages\gcsa\util\date_time_util.py", line 13, in ensure_localisation 
    tz = pytz.timezone(timezone) 
  File "C:\Users\cagatay.guersoy\.conda\envs\env\lib\site-packages\pytz\__init__.py", line 188, in timezone 
    raise UnknownTimeZoneError(zone) 
pytz.exceptions.UnknownTimeZoneError: 'GMT+02:00' 
``` 

**Desktop:**

 - OS:  Windows 10
 - GCSA version: 2.0.0
 - Python version: 3.9.12

**Comment by kuzmoyev** [comment]
Hi, thank you for the issue submission. 
I've pushed a fix in #155. I'll need to test it manually to be sure that this works correctly with Google's API.

I'll publish a new (2.0.1) version when it's tested

**Comment by kuzmoyev** [comment]
Fixed in gcsa=2.0.1. Try:
```
pip install --upgrade gcsa
```
